gr-digital/python/digital/soft_dec_lut_gen.py

```diff
@@ -103,15 +103,27 @@ def soft_dec_table(constel, symbols, prec, npwr=1):
     a bit more expensive to generate the LUT, though it should be
     one-time work.
     '''
-
-    re_min = min(numpy.array(constel).real)
-    im_min = min(numpy.array(constel).imag)
-    re_max = max(numpy.array(constel).real)
-    im_max = max(numpy.array(constel).imag)
+    #normalize constellation to unit power
+    power = 0
+    constel = numpy.array(constel)
+    for point in constel:
+        power += numpy.abs(point)**2
+    amplitude = numpy.sqrt(power/len(constel))
+    constel = constel/amplitude
+    #print("Max soft constellation amp: ", max(abs(constel)), " pwr:", power)
+
+    #padding will simply be 2x the largest re or imag
+    padding = 2
```

Comment:
Just a reminder that if we change any of the C++ code regarding `ptscale` as I have suggested, this Python code probably needs to be updated to do the same thing.